src/v2/controllers/FormController.js

```diff
@@ -109,18 +115,21 @@ export default Controller.extend({
         } else {
           this.showFormErrors(model, error);
         }
+      })
+      .finally(() => {
+        this.toggleFormButtonState(false);
       });
   },
 
   showFormErrors (model, error) {
     //check if error format is an ION response by looking for version attribute. To handle both types of responses.
     if(error.version) {
       const convertedErrors = IonResponseHelper.convertFormErrors(error);
-      model.trigger('error', model, convertedErrors, convertedErrors.responseJSON.errorCauses.length ? false : true);
+      const showBanner = convertedErrors.responseJSON.errorCauses.length ? false : true;
+      model.trigger('error', model, convertedErrors, showBanner);
     } else {
-      model.trigger('error', model, {'responseJSON': error}, true);
+      model.trigger('error', model, {responseJSON: error}, true);
```

Comment:
Is this mostly for LEA or do we have other places where we get back a non ION error response  ?

Reply:
- I'm actually not sure before but wonder if @pradeepdewda-okta aware?
- now there are couple of places where error resp is constructed in this Controller.

Reply:
I see that we are calling showFormErrors above https://github.com/okta/okta-signin-widget/pull/1326/files#diff-cc2782304bc3071a149a654082447a69R96-R99 without setting the version. That might be another use case?

Reply:
this is kind of new case where I'm trying to reuse the `showFormError` method.
I think it was because `showFormErrors` only handle error from `idx.process` may not return API error response but manually crafted error.

Reply:
It's a fall back for LEA cases. I don't know of any other places.